pytoda/datasets/distributional_dataset.py

```diff
@@ -2,24 +2,31 @@
 from typing import Optional, Tuple, Union
 
 import torch
-from torch.distributions.distribution import Distribution
-from torch.random import fork_rng
 from torch.utils.data import Dataset
+from torch.random import fork_rng
 
 from pytoda.types import Any, Tensor
 
 
 @dataclass
-class StochasticItems:
+class StochasticItems(Dataset):
```

Comment:
if we inherit from Dataset, we need to also implement `__len__`. For that I would also pass the dataset length to init.

Reply:
Yikes! no no, this is a residue. I will remove the Dataset.

Reply:
It could be useful actually. You decide.

Reply:
That's true, but we could add this feature when the need arises. For now, it serves its purpose as it is.